covid/transform.py

```diff
@@ -135,6 +137,11 @@
 )
 CDC_CRITERIA_6A_14_DAY_MAX_PERCENT_POSITIVE = "CDC Criteria 6A"
 
+# Policy vs. Trend charts data fields.
+POLICY_VS_TREND_RAW_CASES_PER_MILLION = "Policy vs. Trend Cases per Million (Raw)"
```

Comment:
Do we need `Policy vs. Trend` repeated in all the titles? (This is an honest question!)